roles/ceph-client/tasks/create_users_keys.yml

```diff
@@ -92,3 +92,17 @@
   when:
     - cephx
     - keys | length > 0
+
```

Comment:
@andrewschoen I add some code here for coping ceph.conf and ceph.client.admin.keyring to ansible host, so we could remotely manage all deployed cluster in a single ansible controller.

See reference: https://github.com/kubernetes-incubator/kubespray/pull/2347